docs-2.0/7.data-security/2.backup-restore/1.what-is-br.md

```diff
@@ -23,17 +23,30 @@ This section introduces how backup and restoration are implemented in the BR.
 
 To back up data, the BR sends a backup request to each metad process to trigger the backup process as follows:
 
-1. The SSH login from the BR to the meta and the storage servers is verified. Besides, if remote store such as Amazon S3 or Alibaba Cloud OSS is necessary, their client installation and configuration is verified.
+1. The SSH login from the BR to the meta and the storage servers is verified. Besides, if a remote storage system such as Amazon S3 or Alibaba Cloud OSS is necessary, its client installation and configuration is verified.
```

Comment:
> This step is a pre-request step, instead of an online operation.

NO. checking the SSH login authentication is the first step in the backup or restore process.